### ISSUE TITLE
Timer function? 

### ISSUE THREAD

**Issue by Zeppen** [biemond/com.athom.philipsair#3]
I realized that the air matters app has a feature called timer.
Would be nice to even have that one in the homey app.

**Comment by biemond** [comment]
I don't have this timer mode. so it will be best effort to implement this..
I see the values but please explain how it works and maybe we can do some beta tests.

**Comment by Zeppen** [comment]
best way to explain it should be from the manual of model 3259
![image](https://user-images.githubusercontent.com/1380001/71206629-4e7d3c80-22a5-11ea-8f84-24ed6938a953.png)

**Comment by biemond** [comment]
Added it to latest but it is not yet published. it works for me 

you can set the hours and in the main display you see the remaining minutes.

**Comment by Zeppen** [comment]
> Added it to latest but it is not yet published. it works for me
> 
> you can set the hours and in the main display you see the remaining minutes.

Sweet, I'll give it a try and then I'll update this post 🤓

**Comment by biemond** [comment]
[]()****

**Comment by biemond** [comment]
He,  I will publish it today because just released the latest today. now I can make another test for this. I will update this thread

![Screenshot_20200707-221353](https://user-images.githubusercontent.com/1085004/86838065-0803f980-c0a0-11ea-9b23-288df1591289.png)
![Screenshot_20200707-221427](https://user-images.githubusercontent.com/1085004/86838075-0b978080-c0a0-11ea-95a2-d561cf1eb2f7.png)
![Screenshot_20200707-221452](https://user-images.githubusercontent.com/1085004/86838077-0c301700-c0a0-11ea-85fd-c0342dcd0ce9.png)

**Comment by biemond** [comment]
here we go , this version supports timers https://homey.app/nl-nl/app/com.athom.philipsair/Philips-Air/test/

**Comment by Zeppen** [comment]
> here we go , this version supports timers https://homey.app/nl-nl/app/com.athom.philipsair/Philips-Air/test/

nope doesn't work, looks just the same as i stated earlier.
plus i dont get a confirmation sound in the 3259 as I do with all other commands

**Comment by biemond** [comment]
ok weird, are you using the 2019 version?  let me recheck the code.  indeed you need to hear the sound at least.

**Comment by Zeppen** [comment]
Yup thats correct :) hopefully ill get another one soon since i claimed the garantuee 'cos the fan has started to make some irregular noices at low fan speed. They told me Ill get a new one as a replacement and that i can keep the "faulty" one. I think its only the bearings that are damaged but they have good will 👍 Ill get the same version 3259, do you think it differs on production date?

**Comment by biemond** [comment]
yep, the api can act weird. no errors , success but does no accept it, I saw it before. you will get one with the new firmware.  

can you try this one https://github.com/rgerganov/py-air-control  instead
